tools/ci-ts/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.cjs.json",
+  "compilerOptions": {
+    "outDir": "./dist" /* Redirect output structure to the directory. */
```

Now that i think about this more, we dont need this,
we should be setting noEmit instead. Jest has its own transpile at runtime system
```suggestion
    "noEmit": true
```

---

ahhh right good call :+1: